src/tracks/tracks.c

```diff
@@ -107,7 +107,8 @@ int isStartPointValid(const Track *t) {
    return isValidPoint(&p);
 }
 
-GeoPoint getSectorGeoPointAtIndex(const Track *t, const int index) {
+GeoPoint getSectorGeoPointAtIndex(const Track *t, int index) {
+   if (index < 0) index = 0;
    const int max = isStage(t) ? STAGE_SECTOR_COUNT : CIRCUIT_SECTOR_COUNT;
```

Comment:
So on a circuit track there would be no problem, but on a stage track the sector logic might possibly not work.  Anyway... its good to have.  I'm talking too much and will shut up startiiiiiinnnnngggg now.